modules/tensor_mechanics/src/materials/cohesive_zone_model/CZMComputeDisplacementJumpBase.C

```diff
@@ -85,5 +85,6 @@ CZMComputeDisplacementJumpBase::computeQpProperties()
 void
 CZMComputeDisplacementJumpBase::computeRotationMatrices()
 {
-  _czm_total_rotation[_qp] = RotationMatrix::rotVec1ToVec2(RealVectorValue(1, 0, 0), _normals[_qp]);
+  _czm_total_rotation[_qp] =
+      CohesiveZoneModelTools::computeReferenceRotation(_normals[_qp], _ndisp);
```

Comment:
Is there a better way to get the problem dimension?

Reply:
I could use whatever you prefer, would `_mesh.dimension()` work?